pytorch/inference/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -34,7 +34,6 @@ ARG CUBLAS_VERSION=11.5.1.109
 ARG OMPI_VERSION=4.1.1
 ARG TS_VERSION=0.5.2
 
-ARG CONDA_PREFIX=/opt/conda
```

Comment:
why remove this? isn't the prefix still the same?

comment applies to other dockerfiles as well

Reply:
I discovered a niche issue when installing using an env var set for CONDA_PREFIX for installing mamba that would cause installation to fail. For consistency, I removed all env vars set for CONDA_PREFIX as well as args set for CONDA_PREFIX.